packages/react-components/theme-designer/src/components/Sidebar/Form.tsx

```diff
@@ -43,111 +60,215 @@ const useStyles = makeStyles({
+                />
+                <Input
+                  size="small"
+                  type="number"
+                  min={-50}
+                  max={50}
+                  appearance="outline"
+                  id={sidebarId + 'vibrancy input'}
+                  value={vibrancy.toString()}
+                  onChange={handleVibrancyChange}
+                />
+              </div>
+            </div>
+          </AccordionPanel>
+        </AccordionItem>
+        <AccordionItem value="2">
+          <AccordionHeader>
+            <Caption1Stronger>Step 2 - Accessibility checks</Caption1Stronger>
+          </AccordionHeader>
+          <AccordionPanel className={styles.accordionContainer}>
+            <Switch checked={isDark} onChange={handleIsDarkChange} label={'Dark theme'} />
+            <AccessibilityPanel
+              darkThemeOverrides={darkThemeOverrides}
+              brand={brand}
+              lightThemeOverrides={lightThemeOverrides}
             />
-          </div>
-        </div>
-      </div>
-      <Label htmlFor={sidebarId + 'hueTorsion'}>Hue Torsion</Label>
-      <div className={styles.slider}>
-        <Slider
-          size="small"
-          min={-50}
-          max={50}
-          id={sidebarId + 'hueTorsion'}
-          value={hueTorsion}
-          onChange={handleHueTorsionChange}
-        />
-        <Input
-          size="small"
-          type="number"
-          min={-50}
-          max={50}
-          appearance="outline"
-          id={sidebarId + 'hueTorsion input'}
-          value={hueTorsion.toString()}
-          onChange={handleHueTorsionChange}
-        />
-      </div>
-      <Label htmlFor={sidebarId + 'vibrancy'}>Vibrancy</Label>
-      <div className={styles.slider}>
-        <Slider
-          size="small"
-          min={-50}
-          max={50}
-          id={sidebarId + 'vibrancy'}
-          value={vibrancy}
-          onChange={handleVibrancyChange}
-        />
-        <Input
-          size="small"
-          type="number"
-          min={-50}
-          max={50}
-          appearance="outline"
-          id={sidebarId + 'vibrancy input'}
-          value={vibrancy.toString()}
-          onChange={handleVibrancyChange}
-        />
-      </div>
+          </AccordionPanel>
+        </AccordionItem>
+        <AccordionItem value="3">
+          <AccordionHeader>
+            <Caption1Stronger>Step 3 - Export</Caption1Stronger>
+          </AccordionHeader>
+          <AccordionPanel className={styles.accordionContainer}>
+            <div className={styles.labelName}>
+              <Label htmlFor={themeNameInputId}>Theme name</Label>
+              <Input
+                appearance="outline"
+                id={themeNameInputId}
+                onChange={handleThemeNameChange}
+                placeholder={defaultThemePlaceholderName}
+                value={themeName === defaultThemePlaceholderName ? '' : themeName}
+              />
```

Comment:
Not as necessary as the stickers, but would be nice to use field as well.

Reply:
changed